src/test/java/Homework20.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Reply:
I have deleted the exception now. Please review and grade.


https://github.com/BalaTestPro/A57-QA-Automation-Framework/pull/2